lib/rubocop/config.rb

```diff
@@ -2,15 +2,13 @@
 
 require 'pathname'
 
-# rubocop:disable Metrics/ClassLength
-
 module RuboCop
   # This class represents the configuration of the RuboCop application
   # and all its cops. A Config is associated with a YAML configuration
   # file from which it was read. Several different Configs can be used
   # during a run of the rubocop program, if files in several
   # directories are inspected.
-  class Config
+  class Config # rubocop:disable Metrics/ClassLength
```

Comment:
I tend to prefer the open + close combo if the disable spans several lines. That seems to be what you've gone with in the rest of the cases, too. 🙂